src/mediaplayer.cs

```diff
@@ -1280,12 +1280,17 @@ interface MPVolumeView {
 		[Export ("initWithFrame:")]
 		IntPtr Constructor (CGRect frame);
 
+		[Deprecated (PlatformName.iOS, 13, 0, message: "Use AVRoutePickerView instead")]
```

Comment:
Maybe add a ```.```at the end of the message for consistency.

Reply:
Good catch! Thanks :)